### PR TITLE
Update Canary to v1.5.1

### PR DESCRIPTION
**Pull request by schjonhaug**
## Summary
Update Canary myNode app package to v1.5.1.

## Release summary
Canary v1.5.1 improves self-hosted explorer settings and local platform labels for myNode. It also fixes address-watch syncing edge cases and small self-hosted UI/build issues.

## Upstream release
https://github.com/schjonhaug/canary/releases/tag/v1.5.1

## Test plan
- [x] Built with `mynode-sdk build canary`
- [x] Tested on local myNode

### CODE REVIEW

File: rootfs/standard/usr/share/mynode_apps/canary/scripts/install_canary.sh [hunk 1]
```diff
@@ -9,7 +9,17 @@ set -e
 
 echo "==================== INSTALLING APP ===================="
 
-VERSION="${VERSION:-v1.5.0}"
+VERSION="${VERSION:-v1.5.1}"
+
+remove_canary_images_by_name() {
```
Comment (mynode-dev) [inline rootfs/standard/usr/share/mynode_apps/canary/scripts/install_canary.sh]: This function is replicated twice and should not be necessary. Use the generic remove_docker_images_by_name function.

File: rootfs/standard/usr/share/mynode_apps/canary/scripts/pre_canary.sh [hunk 2]
```diff
@@ -30,5 +35,17 @@ CANARY_SELF_HOSTED_ADMIN_PASSWORD=$(cat "$ADMIN_PASSWORD_FILE")
 JWT_SECRET=$(cat "$JWT_SECRET_FILE")
 EOF
 
+if service_is_enabled mempool; then
```
Comment (mynode-dev) [inline rootfs/standard/usr/share/mynode_apps/canary/scripts/pre_canary.sh]: This should be fine, but it is possible for a service to be disabled or stopped while canary is running which may lead to undesirable behavior.

File: rootfs/standard/usr/share/mynode_apps/canary/scripts/uninstall_canary.sh [hunk 3]
```diff
@@ -8,13 +8,23 @@ set -x
 
 echo "==================== UNINSTALLING APP ===================="
 
+remove_canary_images_by_name() {
```
Comment (mynode-dev) [inline rootfs/standard/usr/share/mynode_apps/canary/scripts/uninstall_canary.sh]: Same here - function is unnecessary

File: rootfs/standard/usr/share/mynode_apps/canary/www/python/canary.py [hunk 4]
```diff
@@ -31,7 +31,6 @@ def canary_page():
         "app_status": app_status,
         "app_status_color": app_status_color,
         "app": app,
-        "canary_username": "admin@local",
```
Comment (mynode-dev) [inline rootfs/standard/usr/share/mynode_apps/canary/www/python/canary.py:34]: Did the app go to not requiring a username?

File: .gitignore [hunk 5]
```diff
@@ -26,3 +26,5 @@ plans/
 releases/
 release.sh
 
+# myNode app build artifacts
+rootfs/standard/usr/share/mynode_apps/*/dist/
```
Comment (mynode-dev) [inline .gitignore]: I don't like this. Apps should not be build within the mynode repo.

**Review by mynode-dev — CHANGES_REQUESTED** [review]

File: rootfs/standard/usr/share/mynode_apps/canary/scripts/pre_canary.sh (hunk 2 above)
Comment (schjonhaug) [reply rootfs/standard/usr/share/mynode_apps/canary/scripts/pre_canary.sh]: Tightened this up. Canary now only writes the local explorer env vars when the myNode service is both enabled and active at Canary startup.

That still leaves runtime changes as startup-scoped behavior: if Mempool or BTC RPC Explorer is stopped/disabled after Canary is already running, restarting Canary refreshes the advertised explorer list. I kept it scoped this way to avoid expanding this PR into a larger dynamic service-status integration.

File: rootfs/standard/usr/share/mynode_apps/canary/scripts/install_canary.sh (hunk 1 above)
Comment (schjonhaug) [reply rootfs/standard/usr/share/mynode_apps/canary/scripts/install_canary.sh]: Fixed. Removed the Canary-specific duplicate helper and switched this back to the existing `remove_docker_images_by_name` helper from `mynode_functions.sh`.

File: .gitignore (hunk 5 above)
Comment (schjonhaug) [reply .gitignore]: Removed. I agree this should not encourage building app artifacts inside the myNode repo. I’ll keep build artifacts outside the upstream repo workflow.

File: rootfs/standard/usr/share/mynode_apps/canary/scripts/uninstall_canary.sh (hunk 3 above)
Comment (schjonhaug) [reply rootfs/standard/usr/share/mynode_apps/canary/scripts/uninstall_canary.sh]: Fixed here too. Removed the duplicate helper and switched uninstall cleanup back to `remove_docker_images_by_name`.

File: rootfs/standard/usr/share/mynode_apps/canary/www/python/canary.py (hunk 4 above)
Comment (schjonhaug) [reply rootfs/standard/usr/share/mynode_apps/canary/www/python/canary.py:34]: Canary still uses the fixed self-hosted username internally (`admin@local`). The app’s self-hosted sign-in flow now submits that fixed username automatically, so the user only needs the generated password. That is why I removed the username from the myNode info page.
